programs/clearing_house/src/state/user.rs

```diff
@@ -176,6 +187,43 @@ impl BankBalance for UserBankBalance {
     }
 }
 
+impl UserBankBalance {
+    pub fn is_available(&self) -> bool {
+        self.balance == 0 && self.open_orders == 0
+    }
+
+    pub fn get_worst_case_token_amounts(
+        &self,
+        bank: &Bank,
+        oracle_price_data: &OraclePriceData,
+    ) -> ClearingHouseResult<(i128, i128)> {
+        let token_amount = get_signed_token_amount(
+            get_token_amount(self.balance, bank, &self.balance_type)?,
+            &self.balance_type,
+        )?;
+
+        let token_amount_all_bids_fill = token_amount
+            .checked_add(self.open_bids)
+            .ok_or_else(math_error!())?;
+
+        let token_amount_all_asks_fill = token_amount
+            .checked_add(self.open_asks)
+            .ok_or_else(math_error!())?;
+
+        if token_amount_all_bids_fill.abs() > token_amount_all_asks_fill.abs() {
+            Ok((
+                token_amount_all_bids_fill,
+                get_token_value(self.open_bids, bank.decimals, oracle_price_data)?,
```

Comment:
shouldnt this be `get_token_value(token_amount_all_bids_fill, bank.decimals, oracle_price_data)?`?